frontend/src/pages/DegreeWizard/DegreeStep/DegreeStep.jsx

```diff
@@ -42,14 +42,10 @@ const DegreeStep = ({ incrementStep }) => {
 
   const searchDegree = (newInput) => {
     setInput(newInput);
-    if (newInput) {
-      const fullDegreeName = Object.keys(allDegrees).map((code) => `${code} ${allDegrees[code]}`);
-      setOptions(
-        fullDegreeName.filter((degree) => degree.toLowerCase().includes(newInput.toLowerCase())),
-      );
-    } else {
-      setOptions([]);
-    }
+    const fullDegreeName = Object.keys(allDegrees).map((code) => `${code} ${allDegrees[code]}`);
+    setOptions(
+      fullDegreeName.filter((degree) => degree.toLowerCase().includes(newInput.toLowerCase())).splice(0, 8),
```

Comment:
Think I already know the answer but why is there an added `splice(0,8)` here? Is this here to restrict the options once we add more degrees?

Reply:
@gtangelo yup